test/Graphics/Image.test.cpp

```diff
@@ -190,44 +190,32 @@ TEST_CASE("[Graphics] sf::Image")
     {
         SECTION("Invalid size")
         {
-            CHECK(!sf::Image({10, 0}, sf::Color::Magenta).saveToFile("test.jpg"));
-            CHECK(!sf::Image({0, 10}, sf::Color::Magenta).saveToFile("test.jpg"));
+            CHECK(!sf::Image({10, 0}, sf::Color::Magenta).saveToFile("test.jpg", sf::Image::SaveFormat::JPG));
+            CHECK(!sf::Image({0, 10}, sf::Color::Magenta).saveToFile("test.jpg", sf::Image::SaveFormat::JPG));
         }
 
         const sf::Image image({256, 256}, sf::Color::Magenta);
 
-        SECTION("No extension")
-        {
-            CHECK(!image.saveToFile("wheresmyextension"));
-            CHECK(!image.saveToFile("pls/add/extension"));
-        }
-
-        SECTION("Invalid extension")
-        {
-            CHECK(!image.saveToFile("test.ps"));
-            CHECK(!image.saveToFile("test.foo"));
-        }
```

Comment:
We could still have an invalid extension test: `CHECK(!image.saveToFile("test", static_cast<sf::Image::SaveFormat>(42)));`